tools/spectral/ipa/rulesets/IPA-104.yaml

```
@@ -38,3 +39,11 @@ rules:
     then:
       field: '@key'
       function: 'getMethodReturnsResponseSuffixedObject'
+  xgen-IPA-104-get-method-response-has-no-input-fields:
+    description: 'The get method response object must not include fields available only on creation or update, ie output fields. http://go/ipa/104'
```

fields available only on creation or update?
 Any way to make it more specific? IT 
```suggestion
    description: 'The get method response object must not include writeOnly (fields that should be used only on creation or update, ie output fields. http://go/ipa/104'
```

---

[Nit] `Get `method

---

Done!